src/components/main/friends/users_list/blocked_list/mod.rs

```diff
@@ -0,0 +1,78 @@
+use dioxus::prelude::*;
+use utils::Account;
+mod blocked_list_tile;
+
+use std::{collections::HashSet, time::Duration};
+
+use crate::components::main::friends::users_list::{
+    blocked_list::blocked_list_tile::BlockedListTile, get_alpha, order_friend_list,
+};
+
+#[derive(Props, PartialEq)]
+pub struct BlockedListProps<'a> {
+    account: &'a Account,
```

Comment:
Although nothing is wrong with it from a glance, but having it as a lifetime isnt needed in this case. It can stay in place though just that its not needed for this field nor does it need to be carried as a reference in the struct. Not a blocking issue though